php/class-plugin-abstract.php

```diff
@@ -182,6 +183,15 @@ public function get_slug() {
 		return $this->slug;
 	}
 
+	/**
+	 * Gets the slug of the post type that stores the blocks.
+	 *
+	 * @return string The slug.
+	 */
+	public function get_post_type_slug() {
+		return $this->post_type_slug;
```

Comment:
This could be moved to the `Util` class?

Inside `Util`, it could just do this:
```
return $this->plugin->post_type_slug;
```

OR we could keep the post type slug in `Block_Post`, and do this instead:

```
return $this->plugin->block_post->slug;
```

The `plugin` property is set automatically when registering any component.

Reply:
Great idea, this is now moved to `Util` in https://github.com/getblocklab/block-lab/pull/384/commits/4614c78849587a6a2dc0b4ab04e287691212979b